redbot/cogs/audio/sql_statements.py

```diff
@@ -366,35 +480,52 @@
     data = excluded.data,
     last_updated = excluded.last_updated;
 """
-LAVALINK_UPDATE = """
+LAVALINK_UPDATE: Final[
+    str
+] = """
 UPDATE lavalink
 SET last_fetched=:last_fetched
 WHERE query=:query;
 """
-LAVALINK_QUERY = """
+LAVALINK_QUERY: Final[
+    str
+] = """
 SELECT data, last_updated
 FROM lavalink
 WHERE
     query=:query
     AND last_updated > :maxage
 LIMIT 1;
 """
-LAVALINK_QUERY_LAST_FETCHED_RANDOM = """
-SELECT data
+LAVALINK_QUERY_ALL: Final[
+    str
+] = """
+SELECT data, last_updated
+FROM lavalink
+"""
+LAVALINK_QUERY_LAST_FETCHED_RANDOM: Final[
+    str
+] = """
+SELECT data, last_updated
 FROM lavalink
 WHERE
     last_fetched > :day
     AND last_updated > :maxage
-ORDER BY RANDOM()
-LIMIT 10
+LIMIT 100
 ;
 """
-LAVALINK_DELETE_OLD_ENTRIES = """
+LAVALINK_DELETE_OLD_ENTRIES: Final[
+    str
+] = """
 DELETE FROM lavalink
 WHERE
-    last_updated < :maxage;
+    last_updated < :maxage
+    LIMIT 100
```

Comment:
`LIMIT` shouldn't be specified in this query.
```suggestion
    last_updated < :maxage
```